specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/examples/ManagedVirtualNetworks_Create.json

```diff
@@ -0,0 +1,36 @@
+{
+  "parameters": {
+    "subscriptionId": "12345678-1234-1234-1234-12345678abc",
+    "resourceGroupName": "exampleResourceGroup",
+    "factoryName": "exampleFactoryName",
+    "managedVirtualNetworkName": "exampleManagedVirtualNetworkName",
+    "ifMatch": null,
+    "managedVirtualNetwork": {
+      "properties": {
+        "description": "A managed Virtual network"
```

Comment:
>description [](start = 9, length = 11)

Same question here

Reply:
Removed the description as it is currently not supported.

Reply:
thanks